### DISCUSSION TITLE
Transform Function Calibration

### DISCUSSION THREAD

**Discussion by jtgilbert** [Riverscapes/riverscapes-tools#785] [VBET]
The data points collected for calibration is plagued with various issues. I have created a subset of the data, cleaned up some outliers, and have working functions, but I don't trust the quality of them very much because of these data issues. 

![calibration](https://user-images.githubusercontent.com/21687931/236263628-93358d7a-c202-4b98-94a5-21b017ba3a5c.png)

The example below shows calibration points over a HAND raster. The labeling of the points is all relative to the larger channel, however the script that pulls the raster values is associating the highlighted point (as an example) with the channel that is right next to it because it falls within that stream segments NHDCatchment feature. So, the value of that point is being associated with a "small" stream, when it actually be associated with the "large" stream that is the mainstem.

I'm not sure what the best path forward is which is why I'm posting here. I (or someone) could take a day going through and creating a subset of validated data with the csv I've already generated. It would obviously consist of much fewer points (this subset currently has ~1800, but they would presumably be meaningful QA'd points.

Thoughts @joewheaton @shelbysawyer @lauren-herbine

**Comment by shelbysawyer** [comment]
Ah, I'm bummed the filtering efforts didn't turn out. I can set aside all day Monday, 5/8 to work on combing through the csv you've already filtered and work on further subsetting them by hand "validating".

Editing to add-- I know we're going to jump on a call later, but what project is this in @jtgilbert?
